test/src/e2e_vm_tests/test_programs/should_pass/dca/revert/src/main.sw

```diff
@@ -0,0 +1,43 @@
+script;
+
+fn revert_01() -> u32 {
+    __revert(0);
```

Comment:
For the completeness and proof, having the same tests for `std::revert` would be good.

Reply:
Ups sorry seen it now! I expected visually to see `std::` :smile: